.circleci/config.yml

```diff
@@ -135,7 +135,7 @@ jobs:
           command: *defaults_Dependencies
       - restore_cache:
           keys:
-          - dependency-cache-5-{{ checksum "src/package.json" }}
+          - dependency-cache-{{ checksum "src/package.json" }}
```

Comment:
Can you make this rely on `package-lock.json`?

Reply:
you are right package-lock.json is a much better source of checksum! fixed!